### ISSUE TITLE
minimum RAM is 16gb?

### ISSUE THREAD

**Issue by longGr** [getsentry/self-hosted#3298]
### Self-Hosted Version

24.1.0

### CPU Architecture

x86_64

### Docker Version

27.1.2

### Docker Compose Version

2.29.1

### Steps to Reproduce

- In the installation instructions the minimum/recommended system requirements are mentioned as 4GB of ram [link to docs](https://develop.sentry.dev/self-hosted/#recommended-system-resource)
- The install script fails when less than 16GB is installed
- the file `install/_min-requirements.sh` had the "soft" requirements removed and the hard requirements increased to 15900MB, was that intended?

### Expected Result

I would expect sentry to not require 16gb of ram for smaller setups. For smaller projects sentry would require way more hardware than the actual project.

### Actual Result

the installation is aborted.

**Comment by aldy505** [comment]
> I would expect sentry to not require 16gb of ram for smaller setups. For smaller projects sentry would require way more hardware than the actual project.

Yes, we're so sorry about the change. Since both Kafka and Clickhouse are required to run Sentry, machines with low RAM may cause both of them to not working at all, this is something we can't really change at the moment.

Although we have an `errors-only` setup, I don't think the RAM can go lower than 16GB though.

We're also updating the docs here, we'll try to get it faster next time: https://github.com/getsentry/sentry-docs/pull/11135

/cc @joshuarli

**Comment by blodone** [comment]
i used to run sentry in a 2GB environment. i don't have millions of errors, i want to track a few errors per month. this is really annoying how bloating sentry got. 

it was a cool tool to track errors but this ecosystem with extensive ram usage kills the whole lightweight feature as it blows up in costs when running in cloud + no more sentry at home at low costs...

**Comment by joshuarli** [comment]
Yep sorry, sentry-docs are quite outdated. See https://github.com/getsentry/sentry-docs/pull/11135. I'll be moving self-hosted README over to official docs website soon.

The fact is Sentry is a complex distributed system and capable product that does a lot more than error monitoring nowadays. I'm sure the errors-only self-hosted can likely get by with 8GB, but you still want some leeway for event processing + python workers when they're actually doing work.

We have had quite a few tickets where strange errors have manifested, all of which likely related to memory running low which can manifest in all kinds of ways. These kinds of tickets take time to respond to, so we've raised the minimum system requirements.
